internal/device/dryRun.go

```diff
@@ -25,13 +51,24 @@ func (b *DryRunBMC) Close(_ context.Context) error {
 
 // GetPowerState returns the device power status
 func (b *DryRunBMC) GetPowerState(_ context.Context) (string, error) {
-	return b.state, nil
+	state, ok := serverStates[b.id]
+	if ok {
+		return state.power, nil
+	}
+
+	return "unknown", nil
 }
 
 // SetPowerState sets the given power state on the device
 func (b *DryRunBMC) SetPowerState(_ context.Context, state string) error {
-	b.state = state
-	return nil
+	serverState, ok := serverStates[b.id]
+	if ok {
+		serverState.power = state
```

Comment:
Given that the `state` argument can be things that are really more _actions_ (like `"cycle"` or `"reset"`) than actual states, should we maybe do some special-casing of those here?

Reply:
Yea could be. Could spin up a go func to change the state after 30 seconds or something if its "reset". IDK. Any suggestions?

Reply:
Or a timer to the state, so if "reset" is called we set the timer, and the next time the state is asked for, if its "reset" we check the timer, if 30 seconds have passed, we change the state to "on"